docs/source/dev-guide/writing-tests/index.rst

```diff
@@ -0,0 +1,77 @@
+===============
+Writing Tests
+===============
+
+This section contains a series of guides and guidelines for writing tests
+in the conda repository.
+
+.. raw:: html
+
+   <hr>
+
+Guides
+======
+
+:doc:`integration-tests`
+This guide gives an overview of how to write integration tests using full
+command invocation. It also covers creating fixtures to use with these types
+of tests.
+
+.. toctree::
+   :hidden:
+   :maxdepth: 1
+
+   integration-tests
+
+.. raw:: html
+
+   <hr>
+
+General Guidelines
+==================
+
+* `Preferred test style (pytest)`_
+* `Organizing tests`_
+* `The "conda.testing" module`_
+* `Adding new fixtures`_
+
+.. note::
+   It should be noted that existing tests may deviate
+   from these guidelines, and that is okay. These guidelines are here to inform how we
+   would like all new tests to look and function.
+
+Preferred test style (pytest)
```

Comment:
```suggestion
Preferred test style (``pytest``)
```

Reply:
@beeankha I originally wanted to do this with the "conda.testing" piece of text too, but I couldn't figure out how to properly escape these characters in the above mini table-of-contents for this section (see line 33). The problem is that the links are wrapped in back ticks and I'm not sure how to include them inside links.

Maybe you know how to do this? 😅

Reply:
This is one of the limitations of RST, seems like. Not sure if what's being talked about [here](https://stackoverflow.com/questions/4743845/format-text-in-a-link-in-restructuredtext) would help, but nested inline markup is not possible without some overengineering, to my knowledge.

Reply:
Thanks for your input @kathatherine, let's just keep it the way it is to  avoid complications.